core/meta/inc/TClass.h

```diff
@@ -232,9 +235,9 @@ friend class TProtoClass;
 
    typedef void (*StreamerImpl_t)(const TClass* pThis, void *obj, TBuffer &b, const TClass *onfile_class);
 #ifdef R__NO_ATOMIC_FUNCTION_POINTER
-   mutable StreamerImpl_t fStreamerImpl;  //! Pointer to the function implementing the right streaming behavior for the class represented by this object.
+   mutable StreamerImpl_t fStreamerImpl; //! Pointer to the function implementing streaming for this class
```

Comment:
Hi Philippe, thanks for reviewing my changes, even though you are still on vacations.

Although you are right, only one of those interpretations makes sense here, i.e., "the class represented by this instance of TClass". If it is for TClass itself, then TClass will be represented by this instance of TClass, and only "this class" does not disambiguate. In any case, I just got annoyed with the extra long lines and created a commit to bring line lengths down to 120 characters. I can drop this commit if it's deemed unnecessary/unwanted.